system/kube-monitoring/values.yaml

```
@@ -16,9 +16,14 @@ global:
   ipmi_service_discovery:
     enabled: false
     configmap_name: ipmi-sd
+    netbox:
+      enabled: false
+      api_token: DEFINED-IN-SECRETS
```

```suggestion
    #api_token:
```
And guard it with a `requires`. If the value is missing by accident we'll use `DEFINED-IN-SECRETS`. Let's not keep on doing that mistake ;)